.github/workflows/vm-nightly-test.yml

```diff
@@ -2,7 +2,10 @@ name: VM Nightly
 on:
   schedule:
     - cron: 0 0 * * *
-
+  push:
```

Comment:
This does not disable the nightly tests right? It just adds it to this specific branch?

Reply:
Right, I'll remove it from our final PR if we end up going with this solution. This was just to allow me to test without running the entire ci suite every time I committed.